### PR TITLE
[Jenkins] Don't build cli on 32-bit Windows builds

### PR DESCRIPTION
**Pull request by bilke**
Follow-up on #1849

### CODE REVIEW

**Comment by endJunction** [comment]
I can imagine, that only `make DataExplorer` will not build the ProcessLib.
Some of the CLI and Utils depend on PL (I see they are switched off), but also unit tests.

**Comment by bilke** [comment]
Yes `make DataExplorer` already worked as expected but I build with `make package` which builds the default targets. Omitting PL and and ML works. See last commit and [Jenkins job](https://jenkins.opengeosys.org/job/ufz/job/ogs/job/PR-1850/3/).

File: CMakeLists.txt [hunk 1]
```diff
@@ -249,12 +249,14 @@ if(TARGET Boost)
     add_dependencies(BaseLib Boost)
 endif()
 add_subdirectory( GeoLib )
-add_subdirectory( MaterialLib )
 add_subdirectory( MathLib )
 add_subdirectory( MeshLib )
 add_subdirectory( MeshGeoToolsLib )
 add_subdirectory( NumLib )
-add_subdirectory( ProcessLib )
+if(OGS_BUILD_CLI OR OGS_BUILD_UTILS OR OGS_BUILD_TESTS)
```
Comment (endJunction) [inline CMakeLists.txt]: Please add a short note, that this is a workaround for the package thing...

**Comment by endJunction** [comment]
Very nice! This reduced the CI time for the 32-bit build from 14 minutes to 3.5 minutes.

Ideas on how to reduce the CI test times (w/o reducing the actual test coverage) are heartily welcome.

**Pull request merged**